CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [0.1.3] -
 
+### Changed
+* Standardized Clustergram code.
```

Comment:
Could you add something a bit more descriptive? IMO the changelog is written for the users of the package. It might be better to write the changes made to the prop names, since this is really the only thing that will be affected.

Reply:
Very good point! Done in f688986.